src/os/bluestore/BlueStore.h

```diff
@@ -1368,10 +1368,10 @@ class BlueStore : public ObjectStore,
     default: return "???";
     }
   }
-  CompressionMode comp_mode = COMP_NONE;      ///< compression mode
+  std::atomic<int> comp_mode;   ///< compression mode
   CompressorRef compressor;
-  uint64_t comp_min_blob_size = 0;
-  uint64_t comp_max_blob_size = 0;
+  std::atomic<uint64_t> comp_min_blob_size;
+  std::atomic<uint64_t> comp_max_blob_size;
```

Comment:
FWIW I think you can still do initalization here with
<pre>
std::atomic<uint64_t> comp_min_blob_size = {0};'
</pre>